soh/soh/SohMenuBar.cpp

```diff
@@ -962,6 +962,10 @@ void DrawEnhancementsMenu() {
             UIWidgets::Tooltip("Always shows dungeon entrance icons on the minimap");
             UIWidgets::PaddedEnhancementCheckbox("Show Gauntlets in First Person", "gFPSGauntlets", true, false);
             UIWidgets::Tooltip("Renders Gauntlets when using the Bow and Hookshot like in OOT3D");
+            if (UIWidgets::PaddedEnhancementCheckbox("Colour Temple of Time's Medallions", "gToTMedallionsColours", true, false)) {
```

Comment:
Lets standardize on "color" instead of "colour" to match the rest of the code base. Both in the menu item name, but also the underlying CVar name introduced in this PR.